gdb-7.8/gdb/breakpoint.c

```diff
@@ -16598,6 +16635,11 @@ initialize_breakpoint_ops (void)
   ops->print_mention = bkpt_print_mention;
   ops->print_recreate = bkpt_print_recreate;
 
+  /* The HSA breakpoint structure that will be hit when the GPU hits a breakpoint. */
+  ops = &hsa_breakpoint_ops;
+  *ops = bkpt_breakpoint_ops;
+  ops->print_it = hsa_bkpt_print_it;
```

Comment:
If we change the name of the structure, then the function should name should also be "gpu_bkpt_trigger_print_it"